NuRadioMC/simulation/simulation.py

```diff
@@ -458,7 +467,7 @@ def run(self):
 
                 ray_tracing_performed = False
                 if(self._station_id in self._fin_stations):
-                    ray_tracing_performed = ('ray_tracing_C0' in self._fin_stations[self._station_id]) and (self._was_pre_simulated)
+                    ray_tracing_performed = self._raytracer.get_raytracing_perfomed() and (self._was_pre_simulated)
```

Comment:
I think this doesn't work yet. Instead you'd need to loop through the ray tracer parameters and check if they are in the hdf5 file.

Reply:
Done